conf/modules/call_repeat_expansions.config

```diff
@@ -52,22 +52,23 @@ process {
             '--write-index=tbi'
         ].join(' ')
         publishDir = [
-            path: { "${params.outdir}/repeat_calling/trgt/single_sample/${meta.id}" },
+            path: { "${params.outdir}/repeats/sample/${meta.id}" },
             mode: params.publish_dir_mode,
             saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
         ]
     }
 
     withName: '.*:CALL_REPEAT_EXPANSIONS:BCFTOOLS_MERGE' {
+        ext.prefix = { "${meta.id}_repeats" }
         ext.args = [
             '--output-type z',
             '--write-index=tbi',
             '--force-single'
         ].join(' ')
         publishDir = [
-            path: { "${params.outdir}/repeat_calling/trgt/multi_sample/${meta.id}" },
+            path: { "${params.outdir}/repeats/family/${meta.id}" },
             mode: params.publish_dir_mode,
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
+            saveAs: { filename -> filename.equals('versions.yml') || !params.skip_repeat_annotation ? null : filename }
```

Comment:
If you have `params.skip_repeat_annotation=true`, will this module run?

Reply:
Yes, it will run but not save files from this process. Instead, annotated files will be saved from the repeat annotation workflow.